src/components/plots/barPlot.ts

```diff
@@ -171,12 +172,14 @@ export module Plots {
       }
     }
 
-    protected _setupDatasetNodes(dataset: Dataset) {
-      super._setupDatasetNodes(dataset);
+    protected _createNodesForDataset(dataset: Dataset) {
+      var drawer = super._createNodesForDataset(dataset);
+      drawer._getRenderArea().classed(Bar._BAR_AREA_CLASS, true);
```

Comment:
That's actually a `d3.Selection` I'm operating on there, so we're stuck with that API.